### PR TITLE
Fix to issue 299 and 297

### PR DESCRIPTION
**Pull request by heylf**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by heylf** [comment]
This PR should fix the issues [297](https://github.com/nf-core/scrnaseq/issues/297) and [299](https://github.com/nf-core/scrnaseq/issues/299)

File: modules/local/mtx_to_seurat.nf [hunk 1]
```diff
@@ -20,9 +20,9 @@ process MTX_TO_SEURAT {
     script:
     def aligner = params.aligner
     if (params.aligner in [ 'cellranger', 'cellrangerarc' ]) {
-        matrix   = "matrix.mtx.gz"
-        barcodes = "barcodes.tsv.gz"
-        features = "features.tsv.gz"
+        matrix   = "filtered_feature_bc_matrix/matrix.mtx.gz"
+        barcodes = "filtered_feature_bc_matrix/barcodes.tsv.gz"
+        features = "filtered_feature_bc_matrix/features.tsv.gz"
```
Comment (grst) [inline modules/local/mtx_to_seurat.nf:25]: This breaks the cellranger workflow. It seems one of the modules outputs these files in a subfolder, while the other does not.

**Review by grst — CHANGES_REQUESTED** [review]

**Comment by grst** [comment]
Now that @fmalmeida set-up nf-test, it would be great to add a testcase for cellranger-arc to make sure such issues don't arise again.

**Comment by heylf** [comment]
> Now that @fmalmeida set-up nf-test, it would be great to add a testcase for cellranger-arc to make sure such issues don't arise again.

I agree. I assigned myself to that issue. I will do it latest before the March Hackathon.

**Comment by grst** [comment]
@heylf, could you please take another look? I'm afraid something got screwed up by merging in the template update.

**Comment by heylf** [comment]
@grst yeah saw right now that this PR was still hanging https://github.com/nf-core/modules/pull/4839. If its merged then it should be fine.

**Comment by grst** [comment]
@heylf, if you look at the diff, there's now basically no changes left since I merged in the template update.

**Comment by heylf** [comment]
@grst Yeah its fine to merge

**Review by grst — APPROVED** [review]
LGTM

**Pull request merged**